expan/core/experiment.py

```diff
@@ -144,3 +144,62 @@ def delta(self, method='fixed_horizon', **worker_args):
 
         result['kpis'] = kpis
         return result
+
+    @staticmethod
+    def _quant_apply(value, threshold):
+        if value < threshold:
+            return False
+        else:
+            return True
```

Comment:
i suppose, can be written simpler:

`def _quant_apply(value, threshold):
      return value > threshold`